app/pages/project/index.cjsx

```diff
@@ -261,13 +271,13 @@ ProjectPage = React.createClass
             Visit {@props.project.display_name}
           </a>
         else
-          <IndexLink to="#{projectPath}" activeClassName="active" className="tabbed-content-tab" onClick={logClick?.bind this, 'project.nav.home'}>
+          <IndexLink to="#{projectPath}" activeClassName="active" className="tabbed-content-tab #{('beta-approved' if betaApproved) ? ''}" onClick={logClick?.bind this, 'project.nav.home'}>
```

Comment:
PFE uses the `classnames` module, which can make logic like this a bit easier to deal with. Have a look at https://www.npmjs.com/package/classnames